models/category/src/types.ts

```diff
@@ -1,9 +1,22 @@
 import type { Category } from '@commercetools/platform-sdk';
 import type { TBuilder } from '@commercetools-test-data/core';
+import type {
+  TClientLoggingGraphql,
+  TLocalizedStringGraphql,
+} from '@commercetools-test-data/commons';
 
 export type TCategory = Category;
 export type TCategoryBuilder = TBuilder<TCategory>;
 export type TCreateCategoryBuilder = () => TCategoryBuilder;
-export type TCategoryGraphql = TCategory & {
+export type TCategoryGraphql = Omit<
+  TCategory,
+  'name' | 'description' | 'createdBy' | 'lastModifiedBy'
+> & {
   __typename: 'Category';
+  // name?: string;
+  // description?: string;
```

Comment:
Is this intentional?

Reply:
Yes, it was meant to show that these props are not available anymore.
I documented this a bit better now: https://github.com/commercetools/test-data/pull/143/commits/3347bd7be3ca6f92f46adb3861c876139ca52ad6